src/diff-snapshot.js

```diff
@@ -219,14 +219,43 @@ function diffImageToSnapshot(options) {
     blur,
     allowSizeMismatch = false,
     comparisonMethod = 'pixelmatch',
+    testPath,
+    currentTestName,
+    runtimeHooksPath,
   } = options;
 
+  const writeFileSync = (pathToFile, content) => {
```

Comment:
Stylistically i'd prefer moving this function out of `diffImageToSnapshot` and renaming to something like `writeFileWithHooks` and have it take `runtimeHooksPath` as a third argument. that way it more clearly differentiates the custom `writeFileSync` from `fs.writeFileSync` 

as an example: 

```suggestion
  const writeFileWithHooks = (pathToFile, content, runtimeHooksPath) => {
```

Reply:
I won't have time to do this modification right now, but I'll do it in the next few days!

Thanks for the review 🙇‍♂️

Reply:
FYI: I'd need 5 arguments as `testPath` & `currentTestName` are also used inside of this `writeFileSync`

Reply:
This is now updated
@JAdshead you can re-review if you want 👌